### PR TITLE
Update Skills Bootcamp Projects schedule

### LINKED ISSUE #648: Introduce npm, ES Lint &c. in the final weeks of the Skills Bootcamp
Earlier than Week 3 of the full-time programme at least!

### PR DESCRIPTION
**Pull request by oliverjam**
- Remove CommonJS module intro from "Node & npm intro" (this will be covered in Server Week)
- Write new guide on Prettier/ESLint/Browsersync (fix #648)
- Add these to SB final 3 weeks schedule

Also relates #698

### CODE REVIEW

File: src/course/syllabus/pre-apprenticeship/projects/project.md [hunk 1]
```diff
@@ -2,7 +2,14 @@ You'll be working on a solo project in your free time for the last 3 weeks. We e
 
 These projects are designed to help you revise the fundamentals you have learnt during the first 9 weeks of the course. We recommend you pick one that involves something you are uncomfortable with. For example if you struggled using fetch and promises then pick a project that will force you to practice that.
```
Comment (Albadylic) [inline src/course/syllabus/pre-apprenticeship/projects/project.md:3]: This is still true but perhaps add a sentence on setting up a dev environment?

File: src/workshops/node-npm-intro/index.md [hunk 2]
```diff
@@ -28,13 +28,17 @@ You'll need to have Node installed on your computer to follow this workshop. Che
 node --version
 ```
 
-You should see a version number printed. Ideally you want at least the current Long-term Support (LTS) version listed on [Node's homepage](https://nodejs.org/en/).
+You should see a version number printed. If you get an error that means Node isn't installed. Follow [our installation guide](/course/handbook/installation/#node) then try again.
 
-If you get an error that means Node isn't installed. Follow [our installation guide](/course/handbook/installation/#node) then try again.
+Our programme relies on some features that were only added in Node version 18 (the current version), so if you have an older version than that you should install the newer one with:
```
Comment (Albadylic) [inline src/workshops/node-npm-intro/index.md:33]: Add this to the installation guide?

**Review by Albadylic — APPROVED** [review]

File: src/course/syllabus/pre-apprenticeship/projects/project.md (hunk 1 above)
Comment (oliverjam) [reply src/course/syllabus/pre-apprenticeship/projects/project.md:3]: I figured that would be obvious from the workshops + technical requirements just below

File: src/workshops/node-npm-intro/index.md (hunk 2 above)
Comment (oliverjam) [reply src/workshops/node-npm-intro/index.md:33]: Yeah good point, will update that in another PR

**Review by sofer — APPROVED** [review]

**Pull request merged**